src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/Common/CommonTypes.Windows.cs

```diff
@@ -2,12 +2,34 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#pragma warning disable CS0618 // Type or member is obsolete
+
 namespace System.Runtime.InteropServices.Tests.Common
 {
     [ComImport]
     [Guid("927971f5-0939-11d1-8be1-00c04fd8d503")]
```

Comment:
I missed this when it was checked in, but with this change it is much more obvious. The `Guid` values should be unquie for all interfaces.

@luqunl what is the behavior if multiple interfaces have the same guid? Is this basically ignored by the type system unless a QI is done?

Reply:
Do you suggest I change the GUID? I actually don't really know a very good list of GUIDs that re valid... any ideas?

Reply:
If you are on Windows, you can generate them quickly from the Visual Studio Developer command prompt. The following command will generate 10 new GUIDs: `uuidgen /c /n10`

Reply:
I would prefer different guid for different types unless there is a special scenario which require to use same guid.